src/odemis/util/dataio.py

```diff
@@ -221,6 +221,7 @@ def open_acquisition(filename, fmt=None):
             data = converter.read_data(filename)
     except Exception:
         logging.exception("Failed to open file '%s' with format %s", filename, fmt)
+        raise
```

Comment:
That's a *big* change. What used to silently fail now passes the "burden" to the caller. There are 53 callers. They all would have to be inspected and checked for handling the exception properly. I would suggest to not do that unless there is a really good reason for it. That said, it might be a good idea to extend the docstring to indicate this behaviour.